translator/tocwconfig/sampleConfig/jmx_eks_config_linux.yaml

```diff
@@ -106,6 +106,450 @@ processors:
                     - kafka.consumer.fetch-rate
         spans: {}
         traces: {}
+    metricstransform/jmx:
+        transforms:
+            - action: insert
+              aggregation_type: ""
+              experimental_match_labels:
+                  code: ^5.*
+              include: apiserver_request_total
+              match_type: regexp
+              new_name: apiserver_request_total_5xx
+              submatch_case: ""
+            - action: update
+              aggregation_type: ""
+              include: tomcat.sessions
+              match_type: ""
+              new_name: ""
+              operations:
+                  - action: aggregate_labels
+                    aggregation_type: sum
+                    experimental_scale: 0
+                    label: ""
+                    label_value: ""
+                    new_label: ""
+                    new_value: ""
+                  - action: delete_label_value
+                    aggregation_type: ""
+                    experimental_scale: 0
+                    label: context
+                    label_value: ""
+                    new_label: ""
+                    new_value: ""
+              submatch_case: ""
+            - action: insert
+              aggregation_type: ""
+              include: DCGM_FI_DEV_FB_USED
```

Comment:
Why are all these GPU metrics actions in the JMX specific metrics transform?

Reply:
Going to look into why they are being added.

Reply:
Adjusted the logic to prevent them from being enabled.